app/src/main/java/com/example/harvestfresh/fragments/EventsFragment.java

```diff
@@ -0,0 +1,232 @@
+package com.example.harvestfresh.fragments;
+
+import android.Manifest;
+import android.app.Dialog;
+import android.location.Location;
+import android.location.LocationRequest;
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+import android.os.Looper;
+import android.text.TextUtils;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Toast;
+
+import com.example.harvestfresh.R;
+import com.google.android.gms.common.ConnectionResult;
+import com.google.android.gms.common.ErrorDialogFragment;
+import com.google.android.gms.common.GooglePlayServicesUtil;
+import com.google.android.gms.location.LocationSettingsRequest;
+import com.google.android.gms.maps.CameraUpdate;
+import com.google.android.gms.maps.CameraUpdateFactory;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.OnMapReadyCallback;
+import com.google.android.gms.maps.SupportMapFragment;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.MarkerOptions;
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+
+
+public class EventsFragment extends Fragment implements OnMapReadyCallback {
+
+    private final static String KEY_LOCATION = "location";
+    private final static int CONNECTION_FAILURE_RESOLUTION_REQUEST = 9000;
+
+    private long UPDATE_INTERVAL = 60000;
+    private long FASTEST_INTERVAL = 5000;
+
+    private SupportMapFragment mapFragment;
+    private GoogleMap newMap;
+    private LocationRequest mLocationRequest;
+
+    Location mCurrentLocation;
+
+    public EventsFragment() {
+        // Required empty public constructor
+    }
+
+    public static EventsFragment newInstance(String param1, String param2) {
+        EventsFragment fragment = new EventsFragment();
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
```

Comment:
Style: If you put one param on another line, put all of them on their own lines, its a bit more legible that way, especially when you add more params in the future.